src/renderer/renderScene.ts

```diff
@@ -104,11 +104,14 @@ export function renderScene(
   }
 
   if (appState.wysiwygElement && appState.wysiwygElement.changeStyle) {
-    appState.wysiwygElement.changeStyle({
-      font: appState.editingElement
-        ? (appState.editingElement as ExcalidrawTextElement).font
-        : "",
-    });
+    if (appState.editingElement) {
+      appState.wysiwygElement.changeStyle({
+        font: (appState.editingElement as ExcalidrawTextElement).font,
+        textAlign: (appState.editingElement as ExcalidrawTextElement).textAlign,
+        color: (appState.editingElement as ExcalidrawTextElement).strokeColor,
+        opacity: (appState.editingElement as ExcalidrawTextElement).opacity,
+      });
+    }
```

Comment:
```suggestion
    if (isTextElement(appState.editingElement)) {
      appState.wysiwygElement.changeStyle({
        font: appState.editingElement.font,
        textAlign: appState.editingElement.textAlign,
        color: appState.editingElement.strokeColor,
        opacity: appState.editingElement.opacity,
      });
    }
```

untested, might need to import the helper stuff